modules/create-kueue-cr.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * install/install-kueue.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-kueue-cr_{context}"]
+= Creating a Kueue custom resource
+
+After you have installed the {kueue-op}, you must create a `Kueue` custom resource (CR) to configure your installation.
+
+.Prerequisites
+
+include::snippets/prereqs-snippet-console.adoc[]
+
+.Procedure
+
+. In the {platform} web console, click *Operators* -> *Installed Operators*.
+. In the *Provided APIs* table column, click *Kueue*. This takes you to the *Kueue* tab of the *Operator details* page.
+. Click *Create Kueue*. This takes you to the *Create Kueue* YAML view.
+. Enter the details for your `Kueue` CR.
++
+.Example `Kueue` CR
```

Comment:
For an example, I would maybe suggest batchJobs as the main one.

This is more "advanced" for Kueue and if we want users to enable Kueue to manage pods, deployments, statefulsets. We need to mention namespace labels and how to safely walk through this.